ext/droption/droption.h

```diff
@@ -184,6 +184,11 @@ class droption_parser_t {
                 ++i; // for last_index
                 break;
             }
+            // Also stop on a non-leading-dash token to support arguments without
```

Comment:
(Optional) Nit: Update docs: (typically this will be options separated by "--" _or when encountering value that are not preceded by a leading-dash token_).

Reply:
Done.